wire-library/wire-runtime/src/jvmTest/kotlin/com/squareup/wire/internal/InternalTest.kt

```diff
@@ -30,4 +30,17 @@ class InternalTest {
     assertEquals(4, countNonNull("xx", "xx", "xx", "xx", null))
     assertEquals(5, countNonNull("xx", "xx", "xx", "xx", "xx"))
   }
+
+  @Test fun sanitizeStrings() {
```

Comment:
Can we move this whole file to `commonTest`?

Reply:
I moved the whole class and seems good